dataform/examples/dataform_datavault/definitions/raw/hub_meter.sqlx

```diff
@@ -0,0 +1,46 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+config {
+  name: "hub_meter",
+  schema: "raw",
+  type: "incremental",
+  uniqueKey: ["meter_id"],
+  bigquery: {
+    partitionBy: "loaddate",
+    clusterBy: ["meter_hash_id", "loaddate"]
+  }
+}
+pre_operations {
+  CREATE TABLE IF NOT EXISTS ${self()} (
+      meter_hash_id STRING,
+      meter_id INTEGER,
+      source STRING,
+      loaddate DATE
+      )
+    PARTITION BY loaddate
+    CLUSTER BY meter_hash_id, loaddate;
```

Comment:
I don't think we need to cluster any more based on the meter_hash_id. We are not filtering based on it. In fact we can remove clustering altogether from here. WDYT?

Reply:
Agree!